src/components/specific/visa/visa-add/visa-add-validator/visa-add-validator-people/VisaAddValidatorPeople.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div
+    class="visa-add-validator-people"
+    v-for="people in peopleList"
+    :key="people.id"
+  >
+    <BIMDataCheckbox
+      :disabled="!people.hasAccess"
+      :modelValue="people.isSelected"
+      @update:modelValue="toggle(people, $event)"
+    >
+    </BIMDataCheckbox>
+    <div
+      class="visa-add-validator-people__picture"
+      :class="{ hide: !people.hasAccess }"
+    >
+      <UserAvatar
+        class="app-header-menu__btn__avatar"
+        :user="people"
+        size="40"
+      />
+    </div>
+    <div v-if="people.fullName" class="visa-add-validator-people__info">
+      <span class="visa-add-validator-people__info__main">{{
+        people.fullName
+      }}</span>
+      <span>{{ people.email }}</span>
+    </div>
+    <div v-else class="visa-add-validator-people__info">
+      <span class="visa-add-validator-people__info__main">{{
+        people.email
+      }}</span>
+    </div>
+    <div
+      v-if="!people.hasAccess"
+      class="visa-add-validator-people__acces-denied"
+    >
+      <div class="visa-add-validator-people__acces-denied">
+        <BIMDataIcon name="warning" size="s" class="fill-high" />
```

Comment:
I think, you want to add `margin="0 6px 0 0"` inside BIMDataIcon to not paste the text to the icon